src/network/webtask.cpp

```diff
@@ -176,13 +174,13 @@ void WebTask::slotStart(int timeoutMillis) {
                 << "Start aborted";
         return;
     }
-    // Task can only be started once
-    m_networkAccessManager = nullptr;
 
     if (m_abort) {
```

Comment:
I think this check is redundant now.

Reply:
No. Can be aborted while being started. I'll add a comment.

Reply:
How? m_abort is only written by a single threads and it was just set.

Reply:
doStart() (template method pattern) is implemented in derived classes and is able to invoke slotAbort() from there. Not likely but still possible.

Reply:
Side-effects could be caused within the same thread during function calls.

Reply:
Ok